### PR TITLE
Onboard devfile organization and integration-tests repo

### PR DESCRIPTION
**Pull request by josephca**
Signed-off-by: josephca joskim@redhat.com

### CODE REVIEW

File: core-services/prow/02_config/_plugins.yaml [hunk 1]
```diff
@@ -3469,6 +3490,33 @@ plugins:
...
+  - retitle
+  - shrug
+  - sigmention
+  - skip
+  - trigger
+  - verify-owners
+  - owners-label
+  - wip
+  - yuks
+  - approve
```
Comment (petr-muller) [inline core-services/prow/02_config/_plugins.yaml:3519]: So you are enabling merge-related plugins like `lgtm` and `approve` but not automated merges themselves. Is that intentional?

File: ci-operator/config/devfile/integration-tests/devfile-integration-tests-master.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-devfile
+  commands: ls -la
+  openshift_installer_src:
+    cluster_profile: aws
```
Comment (petr-muller) [inline ci-operator/config/devfile/integration-tests/devfile-integration-tests-master.yaml:17]: 1. To run e2e tests you're missing one of the stanzas that define which OCP release will be used to install the testing cluster (see https://docs.ci.openshift.org/docs/architecture/ci-operator/#describing-inclusion-in-an-openshift-release)
2. Template based tests are obsolete and you should not add new ones. Use multi-stage tests instead (https://docs.ci.openshift.org/docs/architecture/step-registry/#ci-operator-test-configuration)

Comment (josephca) [reply ci-operator/config/devfile/integration-tests/devfile-integration-tests-master.yaml:24]: Thanks, @petr-muller for your guidance. As a newbie for 'ci-operator',  I actually ran 'make new-repo' command found from [here](https://docs.ci.openshift.org/docs/how-tos/onboarding-a-new-component/#bootstrapping-configuration-for-a-new-repository) and it generated the files for this PR. I think the makefile may be outdated. Anyway, I updated the files according to your suggestion.

Comment (petr-muller) [reply ci-operator/config/devfile/integration-tests/devfile-integration-tests-master.yaml:24]: That's a **great** point! We're in the process of deprecating templates and we will definitely need to update the tool to generate the new content. Thanks for pointing this out!

File: core-services/prow/02_config/_plugins.yaml (hunk 1 above)
Comment (josephca) [reply core-services/prow/02_config/_plugins.yaml:3519]: Correct. I'd like to keep merge as a manual process for now. Thanks,

**Comment by josephca** [comment]
Hi, @petr-muller, could you provide a review? Thanks,

**Review by petr-muller — APPROVED** [review]

**Pull request merged**